.github/workflows/DeployExtensionsProd.yml

```diff
@@ -11,37 +11,41 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2.2.4
       - uses: actions/setup-node@v3
         with:
           node-version: "16.x"
           registry-url: "https://registry.npmjs.org"
-          cache: yarn
+          cache: pnpm
 
       # Ensure everything is compiling
-      - run: "yarn install"
-      - run: "yarn build"
-      - run: "yarn bootstrap"
+      - run: "pnpm install"
+      - run: "pnpm build"
+      - run: "pnpm bootstrap"
 
       # Lets us use one-liner JSON manipulations the package.json files
       - run: "npm install -g json"
 
       # Setup the environment
-      - run: 'json -I -f packages/svelte-vscode/package.json -e "this.version=\`${{ github.ref }}\`.split(\`-\`).pop()"'
+      - run: json -I -f packages/svelte-vscode/package.json -e "this.version=\`${{ github.ref }}\`.split(\`-\`).pop()"
 
-      # To deploy we need isolated node_modules folders which yarn won't do because it is a workspace
+      # To deploy we need isolated node_modules folders which pnpm won't do because it is a workspace
```

Comment:
what exactly is this about?  sounds like `pnpm deploy` could help you. running `npm install` (see below) would get very different versions of the dependencies...

Reply:
Yeah that's intended. We want to get the latest minors of everything we specified

Reply:
When I tried it last time, pnpm deploy still left symlinks in the node_modules folder which the VSIX format can't handle.

Reply:
if we can't use symlinks, then we should use `node-linker=hoisted` instead. or stick with npm .  the way it's done now basically means that whatever we tested is not what gets released. :thinking:

Reply:
i tried to use `pnpm --config.node-linker=hoisted --filter svelte-vscode deploy --prod output-dir` and also tried symlink=false and package import copy, didn't work as intendent. the only other option i can think of with pnpm deploy would be to resolve the symlinks on the fs level after it is done.

Reply:
`node-linker=hoisted` didn't work either for me because. IIRC it was because it's in a workspace, the npmrc doesn't apply if it's not at the workspace root, but putting it in the workspace root had some weird changes that I don't fully remember.

FWIW I looked at how Vue does it and they just bundle it with esbuild instead.

https://github.com/vuejs/language-tools/commit/ab1fafcfc0033f62cdf5104f027d37b9a11d4a14

Reply:
...ooooor we just use npm, as done here, because it works as I want it to

Reply:
You've changed some of the specifiers to use the `workspace:` protocol, does `npm install` resolve them correctly?

Reply:
In the yaml script I modify the package.json to change it back to `*` so npm knows what to do